### ISSUE TITLE
[APP] Splunk Integration for Data Logging

### ISSUE THREAD

**Issue by syariffudinzo** [PipedreamHQ/pipedream#2446]
**Name of app / service**
[Splunk](https://www.splunk.com/) 

**Link to developer documentation**
[Link](https://docs.splunk.com/Documentation)

**Is lack of support preventing you from building workflows, or do you have a workaround?**
Yes, currently our security SOP is to monitor the dataflow from our services to Pipedream, and to ensure that no confidential data is flowing to Pipedream.

**Are there specific actions, or triggers, you'd like to see for this app? Please let us know here or use the Action and Trigger issue templates to open requests for each!**
Any action, trigger from our services to Pipedream is recorded to Splunk, from Splunk we can monitor what is going in and out from our accounts in Pipedream.

**Comment by sergio-eliot-rodriguez** [comment]
Hi @syariffudinzo 

Can you please follow #9743 we are tracking Splunk there

Thanks

**Comment by sergio-eliot-rodriguez** [comment]
Hi @syariffudinzo ,

Apologies for the delay. A [base integration](https://pipedream.com/apps/splunk?via=go) for Splunk has been released. Feel free to check it out.

The Splunk app should work for both Splunk Enterprise and Splunk Cloud: while we didn't get API access to Splunk Cloud using a trial account, I contacted Splunk for a developer license for Splunk Enterprise.

In documentation to work with [Splunk HTTP Event Collector](https://pipedream.com/apps/splunk-http-event-collector), it was mentioned to reach out via email to a Splunk developer support, they were the ones who provided a developer license, good for the entire Splunk Enterprise platform.

There are differences between Splunk Enterprise and Splunk Cloud, see [Access requirements and limitations for the Splunk Cloud Platform REST API](https://docs.splunk.com/Documentation/SplunkCloud/9.1.2308/RESTTUT/RESTandCloud) for more information; the most notable accessing from Pipedream is the need for IP whitelisting, and the need to request Splunk Support to open port 8089 for REST access.

We appreciate your request.
Thank you,
